src/drafts/Tooltip/Tooltip.tsx

```diff
@@ -186,7 +186,7 @@ export const TooltipContext = React.createContext<{tooltipId?: string}>({})
 
 export const Tooltip = React.forwardRef(
   ({direction = 's', text, type = 'description', children, ...rest}: TooltipProps, forwardedRef) => {
-    const tooltipId = useId()
+    const tooltipId = useId(rest.id)
```

Comment:
Wanted to ask, is there a specific reason to not destructure this off of props? It doesn't seem like `id` as an identifier is used and could prevent potential bugs in the future if the position of `{...rest}` changes (like if someone accidentally moves it after where `id={tooltipId}` is set.

Reply:
Great point! I used `rest.id` because id isn't a defined prop like text or children but I guess it doesn't matter though and it is not very safe. Thanks for the comment! I updated it and added tests ✨  https://github.com/primer/react/pull/4200/commits/c93cffe0a6b6f5a58c5429696f336eb927612e70